src/main/java/net/dv8tion/jda/api/entities/ICategorizableChannel.java

```diff
@@ -39,13 +39,17 @@
     /**
      * Get the snowflake of the {@link Category} that contains this channel.
      *
+     * This will return {@code 0} if this channel doesn't have a parent category.
+     *
      * @return The Discord ID snowflake of the parent channel as a long.
      */
     long getParentCategoryIdLong();
 
     /**
      * Get the snowflake of the {@link Category} that contains this channel.
      *
+     * This will return {@code 0} if this channel doesn't have a parent category.
```

Comment:
This is not true as of https://github.com/DV8FromTheWorld/JDA/pull/1993 which will probably cause some conflicts for you anyways.